ow_faults_detection/src/FaultDetector.cpp

```diff
@@ -88,17 +84,20 @@ void FaultDetector::cameraPublishFaultMessages(bool is_fault)
 //// Publish Power Faults Messages
 void FaultDetector::publishPowerSystemFault()
 {
-  ow_faults_detection::PowerFaults power_faults_msg;
-  //update if fault
-  if (m_temperature_fault || m_soc_fault) {
-    //system
-    m_system_faults_flags |= SystemFaultsStatus::POWER_EXECUTION_ERROR;
-    //power
-  } else {
+  // @TODO redundant with system faults publish method,
```

Comment:
I'm going to wait until this gets merged into the epic, and then clean up each fault detection/publishing to use the same scheme with a generic interface. They should all use this state-flag format and then use a generic set header and publish method.